src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -60,6 +66,14 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
   );
 };
 
+ItemUser.propTypes = {
+  photo: PropTypes.string,
+  username: PropTypes.string,
+  followed: PropTypes.any,
+  onPress: PropTypes.func,
+  userid: PropTypes.string,
+  isAnon: PropTypes.string
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The PropTypes have been updated to include `isAnon`. However, `isAnon` is marked as a string, but it seems like it should be a boolean based on its usage in the code. Please correct the PropType for `isAnon`.
```diff
- isAnon: PropTypes.string
+ isAnon: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Ok will do